### ISSUE TITLE
Big Sur compatibility yet? 

### ISSUE THREAD

**Issue by ashleyb1000** [alberti42/iTunes-Volume-Control#33]
Love this little app- but is there any update on compatibility with Mac OS Big Sur yet?

I'm almost waiting for this update before i update to the new OS. 

If anybody can shed some light on this i'd appreciate it!

**Comment by rohanxlr8** [comment]
Works fine for me on Big Sur, it takes a second to load but once loaded no issues

**Comment by ashleyb1000** [comment]
Thanks a lot for the response rohan, will download now- cheers

**Comment by seanbetts** [comment]
I have found that it works ok on Big Sur but regularly crashes/shuts down and I have to constantly restart the programme - anyone else seeing this issue?

**Comment by the3ye** [comment]
Yes, for some reason app closes by itself (disappears from menu bar) and it needs to be restarted. I see it wasn’t updated in months. Is this project closed/abandoned? Any alternatives around?

**Comment by alberti42** [comment]
I am sorry, I did not have time to check in these issues. I am quantum physicist in life, and I did not update to Big Sur yet, because I found for once that Mojave worked really well. Debugging issues on Big Sur would take me to install it on a virtual machine etc.. so these are many hours work and I did not find the time yet. If meanwhile someone wants to suggest a fix through a pull request, I will be happy to update it on GitHub.

**Comment by techtherapy** [comment]
Using BS 11.1 and seems to be working fine. Thanks @alberti42 for this very useful app!

**Comment by alberti42** [comment]
I have posted a newer version compiled for Apple M1 and Intel, https://github.com/alberti42/Volume-Control

Also, I have been running Big Sur since a couple of months. I did not experience any problem since then using my little app. Anyone is still reporting problems with Big Sur?

**Comment by alberti42** [comment]
I would like to draw your attention to the [Volume Control app](https://github.com/alberti42/Volume-Control/), the successor to iTunes Volume Control. 

I recently released version 2.0.1, which has now been notarized by Apple. This makes it straightforward to open and launch.

iTunes Volume Control is no longer supported, so I will close all issues on this repository. If you encounter any new issues with the currently supported app, Volume Control, please open a new issue via this link: https://github.com/alberti42/Volume-Control/issues.
